src/mbgl/sprite/sprite_parser.cpp

```diff
@@ -24,7 +25,11 @@ SpriteImagePtr createSpriteImage(const PremultipliedImage& image,
         ratio <= 0 || ratio > 10 ||
         srcX >= image.size.width || srcY >= image.size.height ||
         srcX + width > image.size.width || srcY + height > image.size.height) {
-        Log::Error(Event::Sprite, "Can't create sprite with invalid metrics");
+        Log::Error(Event::Sprite, "Can't create sprite with invalid metrics: " +
```

Comment:
Okay, got this working with `"%" PRIu32`... https://stackoverflow.com/questions/12120426/how-to-print-uint32-t-and-uint16-t-variables-value